Support/Text/BytePairEncoder.swift

```diff
@@ -97,18 +106,8 @@ public struct BytePairEncoder {
             pairs = (0..<parts.count - 1).map { index in Pair(parts[index], parts[index + 1]) }
         }
 
-        switch variant {
-        case .gpt2:
-            // Encode each token.
-            encodedParts = parts.map({ BytePairEncoder.encodedToken($0) })
-
-        case .roberta, .none:
-            // Encoding has already occurred.
-            encodedParts = parts
-
-        }
         // Check if the new word parts are in the vocabulary, and backtrack if necessary.
-        encoded = encodedParts.flatMap { part -> [String] in
+        let encoded: [String] = parts.flatMap { part -> [String] in
```

Comment:
```suggestion
        let encoded = parts.flatMap { part -> [String] in
```

Again minor, but same deal as above.